settings/src/client.rs

```diff
@@ -33,8 +32,8 @@ pub fn default_config_path() -> String {
 /// ip's within the provided subnet.
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
 pub struct ExitServer {
-    /// Subnet of exits in cluster
-    pub subnet: IpNetwork,
+    /// Ip of exit we first connect to when connected to this cluster
+    pub root_ip: IpAddr,
```

Comment:
So this is the current shared ip across all exits, the multihomed one? Do we need to depend on that or can we just grab an ip from teh list at random to start with?

Reply:
This is the first ip in the cluster that the clients knows about. In order to get a list of ips, the clients needs to know an endpoint address to HTTP to, so it uses this address first. 
This field used to be 'subnet' which was of the form <root_ip>/<sub_len>, but now we just use root_ip